### PR TITLE
Update images.md

### PR DESCRIPTION
**Pull request by DirtySoc**
fix grammar.

### CODE REVIEW

**Comment by DirtySoc** [comment]
signed the CLA.

File: content/en/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ runtime to authenticate to a private container registry.
 This approach is suitable if you can control node configuration.
 
 {{< note >}}
-Kubernetes as only supports the `auths` and `HttpHeaders` section in Docker configuration.
+Kubernetes as is only supports the `auths` and `HttpHeaders` section in Docker configuration.
```
Comment (zacharysarah) [inline content/en/docs/concepts/containers/images.md:97]: I would rephrase this slightly:
```suggestion
Default Kubernetes only supports the `auths` and `HttpHeaders` section in Docker configuration.
```
